packages/jsii-rosetta/test/util.test.ts

```diff
@@ -0,0 +1,63 @@
+import * as ts from 'typescript';
+
+import { annotateStrictDiagnostic, isErrorDiagnostic } from '../lib/util';
+
+const strictBrand = 'jsiiStrict';
```

Comment:
Shouldn't be copied here... instead export the brand in the library, and import it there.